homeassistant/components/ialarmxr/strings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "ialarmxr_no_user_input_data_form": "No user input data form",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "ialarmxr_generic_exception": "iAlarmXR generic error, please read the logs",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
```

Comment:
You can drop the `ialarmxr_generic_exception` and as `unknown` is fine here as there isn't effectively any difference and we want to avoid having to do translations for another string if we don't have to.